ast/ddl.go

```diff
@@ -192,18 +192,27 @@ func (n *DropDatabaseStmt) Accept(v Visitor) (Node, bool) {
 	return v.Leave(n)
 }
 
-// IndexColName is used for parsing index column name from SQL.
-type IndexColName struct {
+// IndexColNameWithExpr is used for parsing index column name from SQL.
+type IndexColNameWithExpr struct {
```

Comment:
Why do you rename it? I think the original name is enough.

Reply:
The original name is `ColName`, but it may be an expression.

Reply:
I think use `IndexColName` is good enough, because these are only one `IndexColName`, this name does not bring any ambiguous.